docs/selectors.md

```diff
@@ -22,6 +22,8 @@ document
   .querySelector('span[attr=value]')
 ```
 
+Selector engine name can be prefixed with `*` to capture element that matches the particular clause instead of the last one. For example, `css=article >> text=Hello` captures the element with the text `Hello`, and `*css=article >> text=Hello` (note the `*`) captures the `article` element that contains some element with the text `Hello`.
```

Comment:
Is it fair to call it the parent element? Since "intermediate" can mean any intermediate result in a string of selectors, I'm wondering if we can be more precise and call it "parent"? I also think parent is easier to understand.